dataherald/sql_generator/__init__.py

```diff
@@ -42,6 +43,21 @@ def format_intermediate_representations(
             )
         return formatted_intermediate_representation
 
+    def format_sql_query(self, sql_query: str) -> str:
```

Comment:
any other part of the code that might need this utility? if so, we should extract it to a sharable utils file either at the root level for all modules or in the sql_generator module